test/syscalls/linux/socket_netdevice.cc

```diff
@@ -178,6 +178,10 @@ TEST(NetdeviceTest, InterfaceMTU) {
   // Check that SIOCGIFMTU returns a nonzero MTU.
   ASSERT_THAT(ioctl(sock.get(), SIOCGIFMTU, &ifr), SyscallSucceeds());
   EXPECT_GT(ifr.ifr_mtu, 0);
+
+  // Check that SIOCSIFMTU succeeds setting with same MTU as retrieved by SIOCGIFMTU.
+  ASSERT_THAT(ioctl(sock.get(), SIOCSIFMTU, &ifr), SyscallSucceeds());
+
```

Comment:
I would also add a test that trying to set anything other than what we just got results in an error on gVisor.

if (IsRunningOnGVisor())  {
   ifr.MTU += 1..
   ...
   ifr MTU -= 2 (..)
  ..
   ifr.MTU = 0
   ...
  
}

Reply:
Add a TODO here to the MTU issue as well.

Reply:
Added both.